src/h5web/vis-packs/core/models.ts

```diff
@@ -46,12 +48,26 @@ export interface AxisConfig {
   flip?: boolean;
 }
 
-export type AxisScale = PickD3Scale<
+export type VisxScaleConfig = PickScaleConfigWithoutType<
   ScaleType.Linear | ScaleType.Log | ScaleType.SymLog | ScaleType.Sqrt,
-  number,
   number
 >;
 
+export interface GammaScaleConfig {
+  domain?: Domain;
+  range?: Domain;
+  exponent?: number;
+  clamp?: boolean;
+}
```

Comment:
I can't use `PickScaleConfigWithoutType<'pow', number>` as `domain` is in this case of type `ContinuousDomain = (number | Date)[]` which is not compatible with `Domain`...